pimd/pim_cmd.c

```diff
@@ -1822,7 +1826,61 @@ DEFUN (show_ip_igmp_join_vrf_all,
 			first = false;
 		} else
 			vty_out(vty, "VRF: %s\n", vrf->name);
-		igmp_show_interface_join(vrf->info, vty, uj);
+		igmp_show_interface_join(vrf->info, vty, uj, GM_JOIN_STATIC);
+	}
+	if (uj)
+		vty_out(vty, "}\n");
+
+	return CMD_SUCCESS;
+}
+
+DEFUN (show_ip_igmp_proxy,
+       show_ip_igmp_proxy_cmd,
+       "show ip igmp [vrf NAME] proxy [json]",
```

Comment:
This could easily be combined with show_ip_igmp_join_cmd since they both do the same thing, just passing a different type.

Reply:
I talked to Jafar about this and he prefers to leave them separate, both for clarity and also because the proxy command may expand in the future. If it does, we'll just have to pull them apart again.